examples/globe_wfs_extruded.html

```diff
@@ -60,10 +89,37 @@
                     globeView.controls.setTilt(45, true);
                 });
             });
-            const d = new debug.Debug(globeView, menuGlobe.gui);
+            var d = new debug.Debug(globeView, menuGlobe.gui);
             debug.createTileDebugUI(menuGlobe.gui, globeView, globeView.wgs84TileLayer, d);
 
-            for (const layer of globeView.getLayers()) {
+            function picking(event) {
+                var htmlInfo = document.getElementById('info');
+                htmlInfo.innerHTML = ' ';
+
+                var intersects = globeView.pickObjectsAt(event, 'WFS Buildings');
```

Comment:
I think `intersects` could be replaced with `picks` or `picked`

Reply:
It's more representative because it returns all the buildings under the mouse, even the hidden buildings, moreover it's the result of a raycaster. Generally picking allows to select a single entity. And the name seems to be consistent with your [last commit](https://github.com/iTowns/itowns/pull/635/commits/b0bbc5e5071c357ae872b45cec731e05df3bbaca#diff-8eb5b5e6b345c723fa6df0ae1c3c7c39R158)

Reply:
Well my last commit was a bad idea.

Here's why: intersects makes sens because you know `pickObjectsAt()` uses raycasting for wfs features. But a regular itowns will not know this - hence my suggestion.